packages/squiggle-lang/__tests__/library/tag_test.ts

```diff
@@ -29,6 +29,15 @@ describe("Tags", () => {
     testEvalToBe("3 -> Tag.hide -> Tag.getHide", "true");
   });
 
+  describe("startOpenToggle", () => {
+    testEvalToBe("3 -> Tag.startOpen -> Tag.getStartOpenState", '"open"');
+    testEvalToBe("3 -> Tag.startClosed -> Tag.getStartOpenState", '"closed"');
+    testEvalToBe(
+      "3 -> Tag.startClosed -> Tag.startOpen -> Tag.getStartOpenState",
```

Comment:
Small feature that one can do "startClosed" then "startOpen", and the last one is what counts. This wouldn't be the case if we saved both separately.